styles/globalStyle.ts

```diff
@@ -57,7 +65,7 @@ const fontStyle = css`
 `;
 
 const bodyStyle = css`
-  ${fontStyle}
+  ${fontStyle};
```

Comment:
The semicolon looks useless.

Reply:
It was added as ESLint highlighted the next line. Removed.